### ISSUE TITLE
on Cancel button animation won't work well

### ISSUE THREAD

**Issue by SanjanaTailor** [4k3R/material-card-animation#2]
When i click on cancel button it reopens it again. I think it can goes back to again on when we click to open that particular screen so can this be fixed ? thanks

Lile on cancel button it should go back to twitter icon with center effect. how to achieve it ?

**Comment by captswag** [comment]
@SanjanaTailor I'm sorry, I don't maintain this anymore.

**Comment by DeevD** [comment]
yes i also too . How to fix this .. cancel button is not work well .. open reveal layout call agian ???

**Comment by DeevD** [comment]
@SanjanaTailor you should put setListener(null) in closeReveal AnimationEnd  like below
 imageButton.animate()
                                .translationX(0f)
                                .translationY(0f)
                                .setDuration(200).setListener(null);
